src/Command/AcquiaCloudCommandBase.php

```diff
@@ -3,6 +3,8 @@
 namespace Acquia\Console\Cloud\Command;
 
 use Acquia\Console\Cloud\Platform\AcquiaCloudPlatform;
+use Acquia\Console\ContentHub\Command\Helpers\PlatformSitesTrait;
```

Comment:
This is a reverse dependency, we should either move it to this library or to helpers.